README.md

```diff
@@ -72,17 +76,111 @@ sha256sum: WARNING: 19 lines are improperly formatted
 ./configure.sh
  ```
 
-**Now you're ready to run the project.**
+7. Steps to generate a Private Key File
 
-*To run from command line:*
+- Using **GenNodeKeyId** class in the project
 
-```bash
-./gradlew run -PmainClass=co.rsk.federate.FederateRunner
+- Set String generator = "pegnatorie1";
+
+- Run the Class to generate a private key
+
+- Create a file **reg1.key** with only the private key as the content
+
+- Add permission to the file by running **chmod 400 reg1.key**
+
+
+8. Creating the signers configuration. **(For Local test / regtest only)**
+
+Create a file **regtest-fed.conf** with the config below
+
+### Signer's configurations
+```
+federator {
+    # keep it false if you don't want to interact with bitcoin
+    enabled = true
+    
+    signers {
+       BTC {
+          type = "keyFile"
+          path = "A/PATH/TO/YOUR/BTC-KEY.key"
+       }
+       RSK {
+          type = "keyFile"
+          path = "A/PATH/TO/YOUR/RSK-KEY.key"
+       }
+       MST {
+          type = "keyFile"
+          path = "A/PATH/TO/YOUR/MST-KEY.key"
+       }
+    }
+    # peers for the bitcoin network
+    bitcoinPeerAddresses = [
+        "127.0.0.1:18332" #bitcoind host and port, change if running on a different port.
+    ]
+}
 ```
+**Note: You can use the same key file for BTC, RSK, and MST**
+
+9. Steps to import and configure the project.
 
 *To import the project to [IntelliJ IDEA](https://www.jetbrains.com/idea/download):*
 
-You can import the project to your IDE. For example, [IntelliJ IDEA Community Edition](https://www.jetbrains.com/idea/). To import go to `File | New | Project from existing sources...` Select `federate-node/build.gradle` and import. After building, run `co.rsk.federate.FederateRunner`
+You can import the project to your IDE. For example, [IntelliJ IDEA Community Edition](https://www.jetbrains.com/idea/). To import go to `File | New | Project from existing sources...` Select `powpeg-node/build.gradle` and import.
+
+- Create a New Application configuration with name **FedRunner**
+
+**VM options**
+
+- -Drsk.conf.file=/<PATH-TO-CONF-FILE>/regtest-fed.conf
+- -Dlogback.configurationFile=/<PATH-TO-LOG-FILE>/logback.xml
+- -Dblockchain.config.hardforkActivationHeights.iris300=0
```

Comment:
Let's just leave only the conf file parameter that it's the only mandatory one. The rest may lead to confussion